assets/css/app.scss

```diff
@@ -0,0 +1,7 @@
+/* This file is for your main application css. */
+body.reddex {
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+@import "navbar";
```

Comment:
Prefer single quoted strings

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#stringquotes)